### DISCUSSION TITLE
How can I make the elements in multiselect rearrange themselves under each other?

### DISCUSSION THREAD

**Discussion by m1x-developer** [mantinedev/mantine#5667] [Q&A]
![image](https://github.com/mantinedev/mantine/assets/83704092/87f9058b-3288-48ec-b61a-4a6d3446dc2e)
![image](https://github.com/mantinedev/mantine/assets/83704092/4f2090e9-1a9a-48a8-a6cf-05ef4b46fb86)

How can I make the elements in multiselect rearrange themselves under each other?     "@mantine/core": "^6.0.15",

**Comment by rtivital** [comment]
I do not get what should be rearranged. The order of the elements in the dropdown is the same as in `data` prop.

**Reply by m1x-developer** [reply]
![image](https://github.com/mantinedev/mantine/assets/83704092/6b85f33c-22c0-49b8-a6c4-557b9e5f6a05)
![image](https://github.com/mantinedev/mantine/assets/83704092/c02bef56-1fee-4f2b-931b-849c4e12398a)

everything is fine with the data, the problem is that the elements do not move under each other, but are stretched to the entire width of the container in the documentation the elements are shifted under each other, but there they are hardcoded to 25rem
I need the elements to rearrange each other regardless of the width, as in the documentation and in screenshot №2.

**Reply by rtivital** [reply]
You need to change tags container flex-direction to column

**Reply by m1x-developer** [reply]
![image](https://github.com/mantinedev/mantine/assets/83704092/1ff714c8-61bd-469d-ba02-842386de4f81)


in this case it looks like this, you need to move them only if there is not enough space

**Reply by rtivital** [reply]
But then it works this way by default. It moves item to the next line only if there is not enough space for them
